dao/src/main/java/cz.muni.fi.pa165.soccermanager/dao/LeagueDaoImpl.java

```diff
@@ -0,0 +1,43 @@
+package cz.muni.fi.pa165.soccermanager.dao;
+
+import cz.muni.fi.pa165.soccermanager.entity.League;
+
+import javax.persistence.EntityManager;
+import javax.persistence.PersistenceContext;
+import javax.persistence.TypedQuery;
+import java.util.List;
+
+public class LeagueDaoImpl implements LeagueDao{
+
+    //@PersistenceContext
+    private EntityManager manager;
+
+
+    @Override
+    public League fetchById(long leagueId) {
+        return manager.find(league.class, leagueId);
```

Comment:
Each class in java starts with upper case -> League.class 
you want to retrieve a 'class' field in class League
with lowwer case it would be a variable and there is no such one -> thats why the build failed

Reply:
[INFO] Reactor Summary:
[INFO]
[INFO] Soccer Manager Parent .............................. SUCCESS [ 6.546 s]
[INFO] Persistent layer ................................... FAILURE [ 9.050 s]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 16.528 s
[INFO] Finished at: 2017-10-25T16:25:06Z
[INFO] Final Memory: 28M/523M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.1:compile (default-compile) on project dao: Compilation failure: Compilation failure:
[ERROR] /home/travis/build/409754/soccer-manager/dao/src/main/java/cz.muni.fi.pa165.soccermanager/dao/LeagueDaoImpl.java:[18,29] cannot find symbol
[ERROR] symbol: class league
[ERROR] location: class cz.muni.fi.pa165.soccermanager.dao.LeagueDaoImpl
[ERROR] /home/travis/build/409754/soccer-manager/dao/src/main/java/cz.muni.fi.pa165.soccermanager/dao/LeagueDaoImpl.java:[23,82] cannot find symbol
[ERROR] symbol: class league
[ERROR] location: class cz.muni.fi.pa165.soccermanager.dao.LeagueDaoImpl